src/holofuel/pages/CreateOfferRequest/CreateOfferRequest.js

```diff
@@ -211,6 +216,12 @@ export function RenderNickname ({ agentId, setCounterpartyNick, setCounterpartyF
           newMessage('This HoloFuel Peer is currently unable to be located in the network. \n Please verify the hash, ensure your HoloFuel Peer is online, and try again after a few minutes.')
           setHasDisplayedNotFoundMessage(true)
         }
+      } else if (id === whoami.id) {
+        setCounterpartyFound(false)
+        if (!hasDisplayedNotFoundMessage) {
+          newMessage('You cannot send yourself TestFuel.')
+          setHasDisplayedNotFoundMessage(true)
+        }
```

Comment:
I think we should do this check in the parent component. We already know our own agentID, so when the counterparty field changes, we can just compare that to our agentId, and show a message then (and disable the button on that condition as well as on `counterpartyNotFound`).

lmk if that's not clear and we can look at it together